src/structures/gdt.rs

```diff
@@ -61,57 +93,69 @@ impl GlobalDescriptorTable {
 }
 
 impl<const MAX: usize> GlobalDescriptorTable<MAX> {
-    /// Creates an empty GDT which can hold `MAX` number of [`Descriptor`]s.
+    /// Creates an empty GDT which can hold `MAX` number of [`Entry`]s.
     #[inline]
     pub const fn empty() -> Self {
         // TODO: Replace with compiler error when feature(generic_const_exprs) is stable.
         assert!(MAX > 0, "A GDT cannot have 0 entries");
         assert!(MAX <= (1 << 13), "A GDT can only have at most 2^13 entries");
+        const NULL: Entry = Entry::new(0);
         Self {
-            table: [0; MAX],
+            table: [NULL; MAX],
             len: 1,
         }
     }
 
     /// Forms a GDT from a slice of `u64`.
     ///
-    /// # Safety
+    /// This method allows for creation of a GDT with malformed or invalid
+    /// entries. However, it is safe because loading a GDT with invalid
+    /// entires doesn't do anything until those entries are used. For example,
+    /// [`CS::set_reg`] and [`load_tss`](crate::instructions::tables::load_tss)
```

Comment:
There are some warnings about unresolved links in the doc comments here.

Reply:
So I'm not sure how to fix this. This method is available with or without the `instructions` feature. But I want to mention functions that only exist if the `instructions` feature exists. This isn't really that big of a problem in practice, but it does produce `rustdoc` warnings when run without the default features.

Reply:
We can suppress that warning.

```rust
#[allow(rustdoc::broken_intra_doc_links)]
```